src/pyedb/dotnet/edb_core/components.py

```diff
@@ -2351,7 +2351,11 @@ def get_pin_from_component(self, component, netName=None, pinName=None):
                 and (self.get_aedt_pin_name(p) in pinName or p.GetName() in pinName)
             ]
         else:
-            pins = [p for p in list(component.LayoutObjs) if int(p.GetObjType()) == 1 and p.IsLayoutPin()]
+            pins = [
```

Comment:
The returned object type is changed. This is not allowed, and it will break customer script. Please revert. Deprecationwarining is added to this method. This method will be removed in the future.